src/error.js

```diff
@@ -358,9 +358,14 @@ export function getErrorReportUrl(message, filename, line, col, error,
   }
   url += '&rt=' + runtime;
 
+  // TODO(erwinm): Remove ca when all systems read `bt` instead of `ca` to
+  // identify js binary type.
   if (isCanary(self)) {
     url += '&ca=1';
   }
+  // Pass binary type.
+  url += '&bt=' + getBinaryType(self);
```

Comment:
Does this need a check to make sure `getBinaryType(self)` is a non-empty value?

Reply:
shouldn't since there is a default value